src/RestWrite.js

```diff
@@ -277,7 +282,8 @@ RestWrite.prototype.runBeforeSaveTrigger = function () {
         updatedObject,
         originalObject,
         this.config,
-        this.context
+        this.context,
+        additionalData
```

Comment:
How about introducing a new `Parse.Cloud.beforeSendVerificationEmail` trigger? That could be a clean separation and we wouldn't need the `additionalData` container anymore.